ci.mk

```diff
@@ -247,5 +247,31 @@ endef
 
 #############################################################################
 
+# Assembler Formatting
+# Provides three make targets:
+#    make asmfmt-check # checks if the provided assembly files are formated correctly
+#    make asmfmt-list # lists assembly files that are not formated correctly
+#    make asmfmt # formats the provided assembly files 
+# @param space-separated list of assembly files
+# @example $(call ci, asmfmt, file1.S fil2.S file3.S)
+
+asmfmt:
+	@asmfmt -w $(_asm_files)
+
+asmfmt-check:
+	@asmfmt -d $(_asm_files)
```

Comment:
I think there is am issue here because asmfmt does not seem to return an error value if the file is not formatted correctly.  A bit of post-processing is needed here: essentially checking if the output of 'asmfmt -d'  is empty.

Reply:
solved here a6dcddeeca59f722923538fd2883ba2afe8f4aa2

Reply:
@josecm Kindly resolve this conversation if the changes attend to your comments. Thank you.